frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+node_modules/
+.pnp/
+.pnp.js
```

Comment:
what is pnp?

Reply:
the gitignore file created with npm create react-app comes in default including those

Reply:
ok, please remove if that's useless